app/models/comment.rb

```diff
@@ -127,6 +127,10 @@ def akismet_attributes
       comment_author_email: comment_owner_email,
       comment_content: comment_content
     }
+
+    attributes[:recheck_reason] = "edit" if edited_at_changed? && comment_content_changed?
```

Comment:
This will be used in the `edited_spam` validation, which uses `will_save_change_to_*` to check whether the attributes are dirty. So I think it would be better to match that, but it is hard to automatically test for what needs to be here...